libs/langchain-google-common/src/chat_models.ts

```diff
@@ -81,14 +85,18 @@ export abstract class ChatGoogleBase<AuthOptions>
 
   safetySettings: GoogleAISafetySetting[] = [];
 
+  safetyHandler;
```

Comment:
I think this should be typed?

Reply:
It didn't _need_ to be, since it inherits from an interface that defines it. But agreed - clarity is good. I'll add it.